build.sh

```diff
@@ -16,9 +16,9 @@ fi
 # not a part of youki workspace. For the reasoning behind this,
 # please check the docs and readme
 
-cargo build --verbose $TGT $1
+cargo build $TGT $1 $2
 cd ./runtimetest
-cargo build --verbose $TGT $1
+cargo build $TGT $1 $2
```

Comment:
I found the output of `--verbose` is too verbose in regular developing activity. I think we can change to use them as needed. e.g.
```
$ ./build.sh
$ ./build.sh --verbose
$ ./build.sh --release
$ ./build.sh --verbose --release
```

a bit dirty but works. we can make this script mature if need later.

Reply:
👍